lib/posix.lua.in

```diff
@@ -519,5 +519,27 @@ if _DEBUG ~= false then
   end
 end
 
+local function prepare_args (pat_mark)
+  local pat, mark = "*", 0
+  if type (pat_mark) == "table" then
+    if type (pat_mark.pattern) == "string" then
+      pat = pat_mark.pattern
+    end
+    if pat_mark.MARK then
+      mark = M.GLOB_MARK
+    end
+  elseif type (pat_mark) == "string" then
+    pat = pat_mark
+  end
+
+  return pat, mark
+end
+
+local cglob = M.glob
```

Comment:
I may be confused, but I don't think that works. Unless you mean that I should require `posix` as a whole into the file? That is, if I just run `posix.glob.glob` now, I don't think there's anything there. Maybe I'm misunderstanding what you mean?